include/ignition/rendering/base/BaseScene.hh

```diff
@@ -522,6 +522,19 @@ namespace ignition
 
       public: virtual void Destroy() override;
 
+      // Documentation inherited.
+      public: virtual void PostRender() override;
+
+      // Documentation inherited.
+      public: virtual void SetCameraPassCountPerGpuFlush(
+            uint8_t _numPass) override;
+
+      // Documentation inherited.
+      public: virtual uint8_t GetCameraPassCountPerGpuFlush() const override;
```

Comment:
nit: ignition omits the `Get` prefix for accessor functions.

So this function name would just be `CameraPassCountPerGpuFlush()`